### DISCUSSION TITLE
Would size should 1 cell be in our game?

### DISCUSSION THREAD

**Discussion by inactivelaylabmanley** [Boxnet-Media/boxnetjam#33] [General]
For like a tile or player or something?

It looks like Pokemon Diamond and such is 32px, but I am not 100% sure.

**Comment by MichaPika** [comment]
> It looks like Pokemon Diamond and such is 32px, but I am not 100% sure.

From what I can tell from researching, they're 16x16 or 8x8 to make a 16x16, I did do limited research though.

I think for the game 16x16 would be okay imo
